### PR TITLE
Fix codespell job

### PR DESCRIPTION
**Pull request by Jainbrt**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
/kind failing-test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: release-tools/SIDECAR_RELEASE_PROCESS.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
     1. Compare the generated output to the new commits for the release to check if
        any notable change missed a release note.
     1. Reword release notes as needed, ideally in the original PRs so that the
-       release notes can be regnerated. Make sure to check notes for breaking
+       release notes can be regenerated. Make sure to check notes for breaking
```
Comment (xing-yang) [inline release-tools/SIDECAR_RELEASE_PROCESS.md:87]: Can you backport the original fix?  Especially for release-tools changes, we need to import them using git subtree.

Comment (Jainbrt) [reply release-tools/SIDECAR_RELEASE_PROCESS.md:87]: @xing-yang I do not have permission to put all the travis workflow related change, can someone help here ?

Error:
 (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/trivy.yaml` without `workflow` scope)

Comment (xing-yang) [reply release-tools/SIDECAR_RELEASE_PROCESS.md:87]: @andyzhangx Are you available to help backport this PR https://github.com/kubernetes-csi/external-snapshotter/pull/1105 to fix the codespell issue?  Thanks!

Comment (andyzhangx) [reply release-tools/SIDECAR_RELEASE_PROCESS.md:87]: I have included all the required changes, and also need to fix a cve in release-8.0: https://github.com/kubernetes-csi/external-snapshotter/pull/1138

**Comment by Jainbrt** [comment]
Closing

**Pull request closed without merging**